app/receipt.go

```diff
@@ -185,9 +185,32 @@ func (app *App) translateCW721Event(ctx sdk.Context, wasmEvent abci.Event, point
 		if tokenID == nil {
 			return nil, false
 		}
+		sender := app.GetEvmAddressAttribute(ctx, wasmEvent, "sender")
+		ownerEvents := GetEventsOfType(response, wasmtypes.EventTypeCW721PreTransferOwner)
+		for _, ownerEvent := range ownerEvents {
```

Comment:
might be good to add a comment here re. why we're doing this - something like
"ERC721 standards dictate that the "sender" in the logs are the actual owner, and in cases where the owner is not the caller, we look for a matching token_id and extract the owner from synthetic events"